bookshelf/1-cloud-run/README.md

```diff
@@ -12,34 +12,41 @@ to below as `MY_BUCKET`.
 
 ### Running Locally
 
-To run locally, update the parameters in `pom.xml`:
+To run your project locally:
 
-* Replace `MY_PROJECT` with your project ID.
-* Replace `MY_BUCKET` with the bucket created above.
+* Set the `BOOKSHELF_BUCKET` environment variable:
 
-Then run:
+      export BOOKSHELF_BUCKET=<YOUR_BUCKET_NAME>
+    
+  Where <YOUR_BUCKET_NAME> is the bucket you created above.
+* Run with the Jetty Maven plugin:
 
-    mvn clean jetty:run-exploded
+      mvn jetty:run-exploded 
 
 **Note**: If you run into an error about `Invalid Credentials`, you may have to run:
 
     gcloud auth application-default login
 
 ### Deploying to Cloud Run
 
-To build your image, update the parameters in `pom.xml` as above, then run:
+To build your image:
 
-    mvn clean package jib:build
-    
-Increase the [memory limit ][configure-memory]for the service:
+* Update the parameters in `pom.xml`:
+  * Replace `MY_PROJECT` with your project ID.
+* Build with [Jib][jib] Maven plugin.
+
+      mvn clean package jib:build
+* Increase the [memory limit ][configure-memory]for the service:
 
-    gcloud beta run services update bookshelf --memory 512M
+      gcloud beta run services update bookshelf --memory 512M
```

Comment:
This can be all one step (plus you need to configure env vars). So, you can shorten to something like this:


* To deploy the app to Cloud Run:
```sh
gcloud beta run deploy bookshelf --image gcr.io/<MY_PROJECT>/bookshelf \
      --platform managed --region us-central1 --memory 512M \
      --update-env-vars BOOKSHELF_BUCKET="<MY_BUCKET>"
```